pkg/cluster/gvik.go

```diff
@@ -0,0 +1,37 @@
+package cluster
+
+import "k8s.io/apimachinery/pkg/runtime/schema"
+
+var (
+	KnativeServingGVK = schema.GroupVersionKind{
```

Comment:
Can it keep naming scheme "Kind" only? So, access is gvk.Kind. I mean, the variable suffix GVK is not needed with the module.

Reply:
> Can it keep naming scheme "Kind" only? So, access is gvk.Kind. I mean, the variable suffix GVK is not needed with the module.

Do you prefer a renaming follow up?

Reply:
did not see this comment, i can remove GVK from the name